app/models/comment.rb

```diff
@@ -1,4 +1,12 @@
 class Comment < ApplicationRecord
   belongs_to :author, class_name: 'User', foreign_key: 'author_id'
```

Comment:
- [ ] Please remove the foreign_key attr from this association because it's already redirecting the process to the `User` model from the class_name attr